### PR TITLE
SCCV Quark - new xenoarch/science shuttle

### PR DESCRIPTION
**Pull request by DreamySkrell**
feedback thread:
  - https://forums.aurorastation.org/topic/21275-sccv-quark-new-xenoarch-shuttle-feedback-thread/#comment-178259

changes:
  - rscadd: "Adds SCCV Quark, new shuttle for xenoarch/science, on Horizon."
  - rscadd: "Small remap of the aux hangar on Horizon, adding some expedition supplies there."
 
reasoning and explanation:
  - The idea is that xenoarchs/science use this if they want to go on a solo adventure for whatever reason (and die to random simplemobs), without stealing the intrepid (that could be used to go to the odyssey scenario site).
  - Xenoarchs would still be allowed to use the intrepid, if it's not used for odyssey or whatever and they want to organize a bigger expedition with more crew (or at least I hope so, I don't make the rules).
  - It's a small shuttle, but it has more than enough seating and some cargo space. I think, at least, it is fine for a xenoarch or two and a few passengers.
  - We need a xenoarch/science shuttle like yesterday, and no one's really shown any interest in mapping it in the past few months, so I made this.
  - If anyone has a different idea and wants to map their own shuttle, it's fine ofc. This is just my PR, anyone is free to open their own, and the best shuttle will be merged

image
image
image
image
image

(explanation on top of description)

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

File: maps/sccv_horizon/code/sccv_horizon_overmap.dm [hunk 1]
```diff
@@ -175,3 +194,42 @@
...
+	desc = "..."
+	shuttle = "Quark"
+	icon_state = "pod" // <-------------- temporary, hopefully someone sprites it some day
+	moving_state = "pod_moving"
+	colors = list("#cfd4ff", "#78adf8")
+	// scanimage = "quark.png" // <-------------- hopefully someone sprites it some day
+	designer = "Hephaestus Industries, NanoTrasen"
+	volume = "14 meters length, 7 meters beam/width, 5 meters vertical height"
+	weapons = "Extruding fore-mounted low-caliber ballistic rotary armament"
+	sizeclass = "Jester-type Scout Skiff"
```
Comment (Greenjoe12345) [inline maps/sccv_horizon/code/sccv_horizon_overmap.dm:214]: The Canary is a Jester class, gonna want adjusting here.

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -1536,6 +1536,27 @@ ABSTRACT_TYPE(/obj/machinery/power/apc)
 	dir = SOUTH
 	pixel_y = -4
 
+/obj/machinery/power/apc/quark
```
Comment (FluffyGhoster) [inline code/modules/power/apc.dm:1539]: This looks like an abstract type and should be marked as such

File: maps/sccv_horizon/code/sccv_horizon_overmap.dm [hunk 3]
```diff
@@ -175,3 +194,42 @@
...
+// 	return skybox_image
+
+/obj/machinery/computer/shuttle_control/explore/terminal/quark
+	name = "\improper Quark control console"
+	shuttle_tag = "Quark"
+	req_access = list(ACCESS_XENOARCH, ACCESS_RESEARCH)
+
+//
+// ------------------------------------------- fin
+//
```
Comment (FluffyGhoster) [inline maps/sccv_horizon/code/sccv_horizon_overmap.dm:235]: Pointless section, we can see it's the end of the file

Comment (DreamySkrell) [reply maps/sccv_horizon/code/sccv_horizon_overmap.dm:235]: awgjtriashjio-0trasemkjhpiea4kt 392 JTO[0I34AJ gy-90h3jw=09HJA3E049I YJHA-0I9EKJ YH-=0923wt j90snj 9054ej=0 y9hj a3e-=0 9jae50-9gjh a=9e 

fine

File: code/modules/power/apc.dm (hunk 2 above)
Comment (DreamySkrell) [reply code/modules/power/apc.dm:1539]: done

File: maps/sccv_horizon/code/sccv_horizon_overmap.dm (hunk 1 above)
Comment (DreamySkrell) [reply maps/sccv_horizon/code/sccv_horizon_overmap.dm]: fixed

**Comment by DreamySkrell** [comment]
!review

**Review by Geevies — APPROVED** [review]
code lgtm

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**